src/screens/FeedScreen.tsx

```diff
@@ -1,35 +1,82 @@
 import React from "react";
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import { SafeAreaView, Text, StyleSheet, ScrollView } from "react-native";
+import {
+  SafeAreaView,
+  Text,
+  StyleSheet,
+  ScrollView,
+  FlatList,
+} from "react-native";
 import { screen_names } from "../constants/ScreenNames";
 import { ParamListBase } from "@react-navigation/native";
 import MainHeader from "../components/layouts/MainHeader";
 import { colors } from "../constants/Colors";
+import FeedsCard from "../components/cards/FeedsCard";
+
+const FEED = [
+  {
+    data: [
+      {
+        title: "Test",
+        body: "Good one",
+        topic: "droidconweb",
+        url: "https://droidcon.co.ke",
+        image: "https://rebrand.ly/5a6672",
+        created_at: "2020-03-19 18:45:49",
+      },
+      {
+        title: "niko fine",
+        body: "this is a test",
+        topic: "droidconweb",
+        url: "https://droidcon.co.ke",
+        image: "https://rebrand.ly/5a6672",
+        created_at: "2020-03-19 18:43:38",
+      },
+    ],
+    meta: {
+      paginator: {
+        count: 2,
+        per_page: "10",
+        current_page: 1,
+        next_page: null,
+        has_more_pages: false,
+        next_page_url: null,
+        previous_page_url: null,
+      },
+    },
+  },
+];
+
+console.log(FEED[0].data);
 
 const FeedScreen = ({
-	navigation,
+  navigation,
 }: NativeStackScreenProps<ParamListBase, screen_names.FEED, undefined>) => {
-	return (
-		<SafeAreaView style={[styles.container, styles.paddingVertical]}>
-			<MainHeader />
-			<ScrollView style={styles.paddingHorizontal}>
-				<Text>Feed screen</Text>
-			</ScrollView>
-		</SafeAreaView>
-	);
+  return (
+    <SafeAreaView style={[styles.container, styles.paddingVertical]}>
+      <MainHeader />
+      <ScrollView style={styles.paddingHorizontal} nestedScrollEnabled={true}>
+        <FlatList
+          data={FEED[0].data}
+          renderItem={({ item }) => <FeedsCard {...item} />}
+          // keyExtractor={({ id }, index) => id}
+        />
+      </ScrollView>
```

Comment:
So the flatlist is vertically scrollable by default. You can check out [React Native Docs](https://reactnative.dev/docs/flatlist) for more information.
In that case, do away with the scrollview the define `styles.paddingHorizontal` on the flatlist.